common/autoware_test_utils/include/autoware_test_utils/autoware_test_utils.hpp

```diff
@@ -387,6 +387,32 @@ T generateTrajectory(
   return traj;
 }
 
+template <>
+inline PathWithLaneId generateTrajectory<PathWithLaneId>(
+  const size_t num_points, const double point_interval, const double velocity,
+  const double init_theta, const double delta_theta, const size_t overlapping_point_index)
+{
+  PathWithLaneId traj;
+
+  for (size_t i = 0; i < num_points; i++) {
+    const double theta = init_theta + i * delta_theta;
+    const double x = i * point_interval * std::cos(theta);
+    const double y = i * point_interval * std::sin(theta);
```

Comment:
minor comments.
These `i`s will trigger `narrowing conversion from size_t` warnings.
I would suggest adding `static_cast` to avoid the warning.

Reply:
Fixed in 98daf521b86a8e505cd68f4ccf9078478c191e26